app/models/user.rb

```diff
@@ -1,15 +1,43 @@
-class User < ApplicationRecord
-  acts_as_tenant(:tenant)
+class User
+  attr_accessor :username
+  attr_accessor :email
+  attr_accessor :first_name
+  attr_accessor :last_name
+  attr_accessor :is_org_admin
 
-  has_many :usergroups, :dependent => :destroy
-  has_many :groups, -> { order(:id => :asc) }, :through => :usergroups
-  validates :email, :presence => true
+  def org_admin?
+    !!is_org_admin
+  end
+
+  def self.find_by_username(username)
```

Comment:
As you can see we'll get some rubocop warnings with this, but it is ok since this is no longer an ApplicationRecord class.

Reply:
@bzwei can this be moved outside of app/models?

Reply:
To me this is still a model, although not a typical AR model.